src/flowTypes.js

```diff
@@ -440,5 +440,5 @@ export type Comment = {
 
 export type Comments = {
     total_count: number,
-    entries: Array<Comment>
+    entries?: Array<Comment>
```

Comment:
Are you sure entries is optional? Based on my testing entries is always returned even if total_count is 0

Reply:
cool, not sure why I changed that